### ISSUE TITLE
accepts background and also background-color

### ISSUE THREAD

**Issue by fera765** [Sharcoux/rn-css#3]
https://github.com/Sharcoux/rn-css/blob/master/src/cssToRN/index.ts

add the code below to accept the use of (background).

```
case 'background':
        Object.assign(result,  { backgroundColor: value })
        break
```

**Comment by Sharcoux** [comment]
How did I miss that?!
Thanks !

**Comment by Sharcoux** [comment]
Fixed in 0.6.9. Thanks
